DDL/StoredProcedures/create_users_table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE PROCEDURE CreateUsersTable
+AS
+CREATE TABLE Users(
+    UserID INT IDENTITY(1,1) PRIMARY KEY, 
+    FirstName VARCHAR(MAX) NOT NULL, 
+    LastName VARCHAR(MAX) NOT NULL, 
+    Username VARCHAR(50) NOT NULL, 
```

Comment:
Please add UNIQUE to Username

Reply:
yep this was done when u requested it, please always check to pull latest changes